test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] As requested earlier, add this file to the ignore file. It should not be present in the repo.
> Also how about adding your test.md file into your .gitignore file as instructed in the project requirement?

Reply:
Hello Roy,
Thanks for your suggestion, I have a doubt, please, let me know if I'm wrong.

- .gitignore file is in the main branch
- test.md is in html-css-files branch

So, when I will merge the PR is when .gitignore will ignore the test.md file?

Please, let me know if I'm wrong

Thanks in advance.

Reply:
Hello, again Roy,

Thanks, I solve it.
I was wrong with my last post, apologize for it. 

Regards,
Alex